.orchestra/config/components/toolchain/lib/gdb.lib.yml

```diff
@@ -27,38 +35,69 @@ configure: |
     --target=(@= triple @) \
     --with-sysroot=$ORCHESTRA_ROOT/(@= triple @) \
     --prefix=$ORCHESTRA_ROOT \
+    --bindir=$ORCHESTRA_ROOT/x86_64-pc-linux-gnu/(@= triple @)/binutils-bin/(@= version @) \
     --datadir=$ORCHESTRA_ROOT/share/binutils-data/(@= triple @)/(@= version @) \
+    --disable-binutils \
+    --includedir=$ORCHESTRA_ROOT/lib64/gdb/(@= triple @)/(@= version @)/include \
     --infodir=$ORCHESTRA_ROOT/share/binutils-data/(@= triple @)/(@= version @)/info \
-    --mandir=$ORCHESTRA_ROOT/share/binutils-data/(@= triple @)/(@= version @)/man \
-    --bindir=$ORCHESTRA_ROOT/x86_64-pc-linux-gnu/(@= triple @)/binutils-bin/(@= version @) \
     --libdir=$ORCHESTRA_ROOT/lib64/binutils/(@= triple @)/(@= version @) \
     --libexecdir=$ORCHESTRA_ROOT/lib64/binutils/(@= triple @)/(@= version @) \
-    --includedir=$ORCHESTRA_ROOT/lib64/gdb/(@= triple @)/(@= version @)/include \
-    --disable-werror \
-    --disable-binutils \
+    --mandir=$ORCHESTRA_ROOT/share/binutils-data/(@= triple @)/(@= version @)/man \
     --disable-etc \
-    --disable-isl \
     --disable-gas \
     --disable-gold \
     --disable-gprof \
-    --disable-ld \
-    --disable-gdbserver \
-    --enable-64-bit-bfd \
     --disable-install-libbfd \
     --disable-install-libiberty \
-    --without-guile \
-    --with-system-readline \
+    --disable-isl \
+    --disable-ld \
+    --disable-shared \
+    --disable-werror \
+    --enable-64-bit-bfd \
+    --enable-gdbserver \
+    --enable-lto \
     --enable-nls \
-    --enable-tui=no \
+    --enable-obsolete \
+    --enable-plugins \
+    --enable-source-highlight \
+    --enable-tui \
+    --with-curses \
+    --with-expat=yes \
     --with-mpfr="$ORCHESTRA_ROOT" \
+    --with-python="$ORCHESTRA_ROOT/bin/python3" \
+    --with-system-gdbinit="$ORCHESTRA_ROOT/(@= triple @)/etc/gdb/gdbinit" \
+    --with-system-readline \
+    --without-guile \
     --without-included-gettext \
-    --disable-shared \
-    --enable-obsolete \
     --without-stage1-ldflags \
     CFLAGS="-w -ggdb3 -O3" \
     CXXFLAGS="-w -ggdb3 -O3"
-build_system: make
+install: |
+  cd "$BUILD_DIR"
+  (@= make @)
+  (@= make @) install DESTDIR="$DESTDIR"
+
+  GDBINIT="$DESTDIR$ORCHESTRA_ROOT/(@= triple @)/etc/gdb/gdbinit"
+  mkdir -p "$(dirname "$GDBINIT")"
+
+  cat >"$GDBINIT" <<'EOF'
+  set pagination off
+  EOF
+
+  (@- if triple.startswith("x86_64"): @)
+  echo "set disassembly-flavor intel" >> "$GDBINIT"
+  (@ end @)
+
+dependencies:
+  - expat
+  - gmp
+  - ncurses
+  - python
+  - readline
+  - source-highlight
 build_dependencies:
+  - host-c-toolchain
+  - host-cxx-toolchain
```

Comment:
One implies the other. If `gdb` is not written in C++, don't use `host-cxx-toolchain`. Even if source-highlight is in C++, it'll be its own responsibility to pull in the dependency.